packages/builder/src/definition.ts

```diff
@@ -25,15 +25,18 @@ export type StateLayers = {
   };
 };
 
+/**
+ * Cannonfile metadata layout
+ */
 export class ChainDefinition {
   private raw: RawChainDefinition;
 
   readonly allActionNames: string[];
 
-  // actions which have no dependencies
+  /** actions which have no dependencies */
```

Comment:
is jekyll not compatible with `//` and you have to use `/** */`?

Reply:
No this is to enable TSDOC comments which help with documenting certain things. Also have some useful features like being able to see these comments on instances of this class.